build/steps-build-ios.yml

```diff
@@ -32,6 +32,16 @@ steps:
 
 - template: templates/gitversion.yml
 
+- task: PowerShell@2
```

Comment:
I would suggest to create a template task for the build number. To avoid duplicating code